Jenkinsfile_CNP

```diff
@@ -91,10 +92,8 @@ withPipeline(type, product, component) {
       env.TEST_STORAGE_ACCOUNT_URL = "https://${env.TEST_STORAGE_ACCOUNT_NAME}.blob.core.windows.net"
       env.TEST_STORAGE_ACCOUNT_KEY = previewAccountKey(kubectl, storageSecret, namespace)
 
-      env.STORAGE_BULKSCAN_CONNECTION_STRING = storageAccountConnectionString(
-        previewAccountName(kubectl, bulkScanStorageAccountSecret, namespace),
-        previewAccountKey(kubectl, bulkScanStorageAccountSecret, namespace)
-      )
+      def bulkScanStorageAccountName = previewAccountName(kubectl, bulkScanStorageAccountSecret, namespace)
+      env.STORAGE_BULKSCAN_URL = "https://${bulkScanStorageAccountName}.blob.core.windows.net"
```

Comment:
for preview there is no app gw as we use osba so existing config should work ?

Reply:
but do we want different behaviour for preview?

Reply:
it has to be for now as we don't have app gw for preview (this is what we have in processor)

Reply:
This is the storage account's URL

Reply:
needs discussion

Reply:
apologies this is correct